.github/pull_request_template.md

```diff
@@ -1,8 +1,8 @@
 ## Checklist before submitting
 
 - [ ] Did you read the [contributor guide](https://github.com/open-ce/open-ce/blob/main/CONTRIBUTING.md)?
-- [ ] Did you update any affected [documentation](https://github.com/open-ce/open-ce/blob/main/doc/)?
-- [ ] Did you write any [tests](https://github.com/open-ce/open-ce/blob/main/tests/) to validate this change?  
+- [ ] Did you update any affected [documentation](https://github.com/open-ce/open-ce-builder/blob/main/doc/)?
+- [ ] Did you write any [tests](https://github.com/open-ce/open-ce-builder/blob/main/tests/) to validate this change?  
```

Comment:
We're removing this file, but we should probably make these changes in https://github.com/open-ce/.github